### DISCUSSION TITLE
Endpoint usando o verbo Patch

### DISCUSSION THREAD

**Discussion by Douglasproglima** [balta-io/forum#593] [Tira Dúvidas] [answered]
Olá, gostaria como posso implementar um endpoint usando o método PATCH. Estou com dúvidas pois sabemos que a lib System.Text para serializar, deserializar objetos em json é mais eficaz do newtonsoftjson, em várias abordagens hoje explicadas na internet um endpoint composto pelo verbo PATCH recebe um parâmetro do tipo JsonPatchDocument<DTO>, dai fazemos referente a lib newtonsoftjson.

Exemplo:
![image](https://github.com/balta-io/forum/assets/9982359/63c6e017-6b3a-4a52-8bb2-4b922ae336ab)

Qual forma mais eficiente atualmente para criarmos um endpoint de atualização parcial no .NET

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Douglasproglima, como vai?

Até onde sei, não temos problemas com o Patch faz um tempo já:
https://github.com/dotnet/aspnetcore/issues/36198

Segue exemplo que rodei aqui e funcionou:
```csharp
app.MapPatch("/patch", (MyModel model) => Results.Ok((object?)model));

public record MyModel(int Id, string Text);
```
